### PR TITLE
fix: NTRN-278. Fix chains startup to work with interchain security

### PR DESCRIPTION
**Pull request by albertandrejev**
Make chains startup to work with latest version of neutron with CCV modules.

Related PRs:
- https://github.com/neutron-org/neutron-integration-tests/pull/46
- https://github.com/neutron-org/neutron-contracts/pull/67

### CODE REVIEW

File: network/init.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ADMIN_ADDRESS=neutron14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9s5c2epq
 BINARY=neutrond
+GAIAD_BINARY=gaiad
```
Comment (foxpy) [inline network/init.sh]: Let's rename BINARY → NEUTRON_BINARY as well?

Comment (foxpy) [reply network/init.sh]: or maybe even NEUTRON**D**_BINARY :thinking:

**Review by zavgorodnii — APPROVED** [review]
tested

File: network/dockerbuilds/Dockerfile.gaia [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
```
Comment (swelf19) [inline network/dockerbuilds/Dockerfile.gaia:1]: don't exactly understand, where do we use this file?

File: network/init.sh [hunk 3]
```diff
@@ -96,8 +96,6 @@ sed -i -e 's/prometheus-retention-time = 0/prometheus-retention-time = 1000/g' $
 
 sed -i -e 's#"tcp://0.0.0.0:26656"#"tcp://0.0.0.0:'"$P2PPORT_2"'"#g' $CHAIN_DIR/$CHAINID_2/config/config.toml
 sed -i -e 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:'"$RPCPORT_2"'"#g' $CHAIN_DIR/$CHAINID_2/config/config.toml
-sed -i -e 's/timeout_commit = "5s"/timeout_commit = "1s"/g' $CHAIN_DIR/$CHAINID_2/config/config.toml
```
Comment (swelf19) [inline network/init.sh:99]: what is the purpose of increasing timeouts? just make chains block time differs?

File: Dockerfile [hunk 4]
```diff
@@ -1,3 +1,12 @@
+# syntax=docker/dockerfile:1
```
Comment (swelf19) [inline Dockerfile:1]: seems like we do not use `start-cosmopark` from this repo, our cosmopark configuration moved to integration tests. May be we can get rid of the dead code?

**Review by swelf19 — APPROVED** [review]

File: Dockerfile (hunk 4 above)
Comment (zavgorodnii) [reply Dockerfile:1]: this actually makes sense

File: network/init.sh (hunk 3 above)
Comment (foxpy) [reply network/init.sh:99]: IMO, having different block times might expose some races in contracts, so maybe we should have increased timeout? (I cannot prove what I am saying, I am just speculating).

File: Dockerfile (hunk 4 above)
Comment (zavgorodnii) [reply Dockerfile:1]: we'll do that is a separate pr

File: network/init.sh (hunk 3 above)
Comment (albertandrejev) [reply network/init.sh:99]: The reason why I used different block time for different parts of the infrastructure is to have environment similar to real world.

**Pull request merged**